pkg/applicationserver/io/web/webhooks.go

```diff
@@ -268,7 +268,7 @@ func (w *webhooks) NewSubscription() *io.Subscription {
 			case <-w.ctx.Done():
 				return
 			case msg := <-sub.Up():
-				if err := w.handleUp(w.ctx, msg); err != nil {
+				if err := w.handleUp(msg.Context, msg.ApplicationUp); err != nil {
```

Comment:
This for example could be problematic, since `msg.Context` may already be canceled at this point.